src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1701162864.inc.php

```diff
@@ -0,0 +1,146 @@
+<h1>Build #1701162864</h1>
+<h2>Date: 2023-11-28</h2>
+<div class="changelog">
+    <ul>
+        <li>- ref #832: add preventReferenceCopy to property field ("Eigenschaften")</li>
```

Comment:
A boolean value should in general always express the positive, not the negative. So I would prefer "allowReferenceCopy" with a default of true. But I also see why this was done this way here, so no need for a change